### ISSUE TITLE
custom springConfig not working

### ISSUE THREAD

**Issue by alex-cory** [Temzasse/react-modal-sheet#28]
See the [codesandbox](https://codesandbox.io/s/react-modal-sheet-springconfig-bug-4e2m3)

**Comment by Temzasse** [comment]
Hi @alex-cory 👋🏻 

Good catch! 👏🏻 The spring config is only used for the snapping behaviour and I forgot to add it to the `AnimatePresence` related animation. This should be an easy fix in `SheetContainer`:

```js
animate={{
  y: initialY,
  transition: { type: 'spring', ...springConfig },
}}
```

**Comment by Temzasse** [comment]
This is fixed in `v1.4.0` ✌🏻
